### PR TITLE
Clinic finder list

### PR DESCRIPTION
**Pull request by Buhle79**

### CODE REVIEW

File: clinicfinder/views.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+
+class ClinicFinderView(generics.GenericAPIView, ListModelMixin):
+    serializer_class = LocationSerializer
+    permission_classes = (permissions.IsAuthenticated,)
+
+    def get(self, request, *args, **kwargs):
+        distance_range = settings.LOCATION_SEARCH_RADIUS
+        params = request.query_params
+        if "longitude" in params and "latitude" in params:
```
Comment (erikh360) [inline clinicfinder/views.py:20]: You can use the serializer for this validation

**Review by erikh360 — COMMENTED** [review]
Just the one comment

Just confirm with SRE if the DB changes you made will work, the postgis

@bearnard  is aware of the changes

File: tbconnect/migrations/0001_initial.py [hunk 2]
```diff
@@ -1,12 +1,11 @@
-# Generated by Django 3.1 on 2020-08-25 14:07
+# Generated by Django 3.1.14 on 2022-06-13 07:12
```
Comment (bearnard) [inline tbconnect/migrations/0001_initial.py]: hmm, why is there a change in migrations that have already run?

Comment (Buhle79) [reply tbconnect/migrations/0001_initial.py]: My local project broke then i had to fix it by deleting migrations then the new initial file was created.

Comment (bearnard) [reply tbconnect/migrations/0001_initial.py]: We'll need to try fix this because this as is will fail in qa/prod or for any existing db.

Comment (bearnard) [reply tbconnect/migrations/0001_initial.py]: My suggestion is to either revert that commit if it exists or manually make this file match what's in `develop` and then re-run `makemigrations` to get a new migration for your model additions.

File: tbconnect/models.py [hunk 3]
```diff
@@ -107,11 +120,15 @@ class TBCheck(ExportModelOperationsMixin("tb-check"), models.Model):
     commit_get_tested = models.CharField(
         max_length=3, choices=COMMIT_CHOICES, null=True, blank=True
     )
-    research_consent = models.BooleanField(null=True)
+    research_consent = models.BooleanField(null=True, blank=True)
```
Comment (bearnard) [inline tbconnect/models.py]: why blank=true?

Comment (Buhle79) [reply tbconnect/models.py]: It's was part of reverted migrations.

Comment (bearnard) [reply tbconnect/models.py]: I'm more interested to know why we allow it to be anything other than True/False/null ?

Comment (Buhle79) [reply tbconnect/models.py]: For miners they won't get the research consent question so their feedback will be None

**Comment by bearnard** [comment]
👍

**Pull request merged**